src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -119,6 +119,7 @@ def load_command_table(self, _):
     with self.command_group('openshift', openshift_managed_clusters_sdk,
                             client_factory=cf_openshift_managed_clusters) as g:
         g.custom_command('create', 'openshift_create', supports_no_wait=True)
+        g.custom_command('update', 'openshift_update', supports_no_wait=True)
```

Comment:
Do we have a `wait` command as there are so many `support_to_wait` commands ?

Reply:
@arrownj yes we do have `wait` command: https://github.com/Azure/azure-cli/blob/67d91648c5cf92cd560db06e1bc62cdcf7480004/src/azure-cli/azure/cli/command_modules/acs/commands.py#L126